src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVIntervalTree.java

```diff
@@ -0,0 +1,904 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
```

Comment:
There was a year long argument about this and the gatk closed intervals....  @droazen still bears the scars of the interval wars.